byzcoin/api.go

```diff
@@ -417,3 +450,54 @@ func DefaultGenesisMsg(v Version, r *onet.Roster, rules []string, ids ...darc.Id
 	}
 	return &m, nil
 }
+
+func verifyGenesisBlock(sb *skipchain.SkipBlock, req *CreateGenesisBlock) error {
```

Comment:
nit: I would name those parameters like: `actual *skipchain.SkipBlock, expected *CreateGenesisBlock`, so it's obvious which input is trusted.